bundle/compliance/policy/azure/data_adapter.rego

```diff
@@ -1,11 +1,19 @@
 package compliance.policy.azure.data_adapter
 
-# import data.compliance.lib.common
-
 resource = input.resource
 
 properties = resource.properties
 
+activity_log_alerts = resource.activity_log_alerts
+
 is_storage_account {
 	input.type == "azure-storage-account"
 }
+
+is_subscription_resource {
+	input.type == "azure-subscription"
+}
+
+is_activity_log_alerts {
+	input.type == "azure-activity-log-alert"
```

Comment:
https://github.com/elastic/cloudbeat/pull/1372/files#diff-765fedb565916ef36d9d870ba1c3c6e58041c73dc8913c6bb92c5c06fa39d21fR80
The resource type is `azure-activity-log-alerts` and not `azure-activity-log-alert`

Reply:
Committed code that will be according to this type.
Ignore comment.